docs/content/stable/architecture/transactions/read-committed.md

```diff
@@ -1565,7 +1565,10 @@ Read Committed interacts with the following feature:
 
     For more details, see [#12494](https://github.com/yugabyte/yugabyte-db/issues/12494).
 
-* Read restart and serialization errors are not internally handled in read committed isolation if the query's response size exceeds the YB-TServer GFlag `ysql_output_buffer_size`, which has a default value of 256KB (see [#11572](https://github.com/yugabyte/yugabyte-db/issues/11572)).
+* Read restart and serialization errors are not internally handled in read committed isolation:
+- if the query's response size exceeds the YB-TServer GFlag `ysql_output_buffer_size`, which has a default value of 256KB (see [#11572](https://github.com/yugabyte/yugabyte-db/issues/11572)).
+- if multiple semi-colon separated statements in a single query string are sent via the simple query protocol (see [#21833](https://github.com/yugabyte/yugabyte-db/issues/21833)).
+- for statements other than the first one in a batch sent by the driver (see [#21607](https://github.com/yugabyte/yugabyte-db/issues/21607))
```

Comment:
The GHI referenced is slightly off from what the sub-bullet states. I think it should say "except for [GHI] currently" or something like that. The other bugs referenced are for when the bug is fixed the sub-bullet would go away.

Reply:
done